### ISSUE TITLE
Docs building is broken

### ISSUE THREAD

**Issue by mathbunnyru** [mamba-org/mamba#3585]
### Troubleshooting docs

- [X] My problem is not solved in the Troubleshooting docs

### Anaconda default channels

- [X] I do NOT use the Anaconda default channels (pkgs/* etc.)

### How did you install Mamba?

Mambaforge or latest Miniforge

### Search tried in issue tracker

docs, sphinx

### Latest version of Mamba

- [X] My problem is not solved with the latest version

### Tried in Conda?

Not applicable

### Describe your issue

It seems that for a while docs building has been broken.
https://app.readthedocs.org/projects/mamba/builds/

The latest successful build was on 7th October.
It is also the day when Read the Docs Addons were enabled by default: https://about.readthedocs.com/blog/2024/07/addons-by-default/

**Comment by mathbunnyru** [comment]
I also think the RTD should be configured to be run in branches: https://docs.readthedocs.io/en/stable/pull-requests.html
As far as I remember the admin of RTD has to click one checkbox in the Admin page - it makes the dev's life so much easier when there are documentation changes.

**Comment by mathbunnyru** [comment]
I think this is a major issue, and it's not good that there is no automatic checking the docs are fine

**Comment by mathbunnyru** [comment]
I tried to start contributing to this project, for example, and docs mention `Taskfile`: https://mamba.readthedocs.io/en/latest/developer_zone/dev_environment.html

But it has been removed 3 weeks ago: https://github.com/mamba-org/mamba/pull/3544

**Comment by mathbunnyru** [comment]
I think I found the reason.
7th October is when Python 3.13 was released and `imghdr` has been removed in this release.
 
For some reason, rtd installs python 3.13, but also installs pretty old sphinx 5.3, which uses `imghdr` somewhere in its internals.

**Comment by mathbunnyru** [comment]
I've also checked how to enable building docs in PRs.
Just go to https://readthedocs.org/dashboard/mamba/edit/ (you need to be logged in and have admin access to RTD project, I don't have it), click near `Build pull requests for this project`, and then `Save.

This is how it's going to look like in PRs: https://github.com/mathbunnyru/mamba/pull/1

**Comment by mathbunnyru** [comment]
> I've also checked how to enable building docs in PRs. Just go to https://readthedocs.org/dashboard/mamba/edit/ (you need to be logged in and have admin access to RTD project, I don't have it), click near `Build pull requests for this project`, and then `Save.
> 
> This is how it's going to look like in PRs: [mathbunnyru#1](https://github.com/mathbunnyru/mamba/pull/1)

@Hind-M I highly recommend you doing this, it will prevent this problem from happening again and it greatly helps to review docs PR changes (you can inspect not just the source code, but the built pages as well).

**Comment by Hind-M** [comment]
Yes absolutely, I will pass the info to the admin people who have the rights on RTD, thanks!
